src/DelegationManagement.sol

```diff
@@ -10,12 +10,12 @@
  *
  */
 
-pragma solidity >=0.6.0 <=0.8.0;
+pragma solidity >=0.8.0;
```

Comment:
I would recommend having this stated in the description. There are multiple PRs already changing the solidity version and they should be separated if possible from the action it is trying to achieve. I honestly was hoping for more active engagement from the creator. Because even if not all of the minor improvements I suggested in PR #1 were to pass, they could've saved a lot of headaches to the following PRs I've been seeing here

Reply:
Or if your code works without changing the compiler version, it would be best.

Reply:
Well, the creators will get hacking post new year's eve as stated in the tweet i believe.